src/twisted/internet/defer.py

```diff
@@ -2248,6 +2251,17 @@ def loadData(url):
     reaching its end, a C{return} statement, or by calling L{returnValue}).
     A C{CancelledError} will be raised from the C{yield}ed L{Deferred} that
     has been cancelled if that C{Deferred} does not otherwise suppress it.
+
+    C{inlineCallbacks} behaves very similarly to coroutines. Since Twisted NEXT
+    it is possible to rewrite functions using C{inlineCallbacks} to C{async def}
+    in piecewise manner and be mostly compatible to existing code.
+
+    The rewrite process is simply replacing C{inlineCallbacks} decorator with
```

Comment:
Minor comment.

I am not sure if we need this paragraph. 
I would say that this info can be part of `defer-intro.rst`

Otherwise we end up with duplicated documentation.

Reply:
I think that docstrings and defer-intro.rst serve different purposes. The former is a reference, a person knows what he wants to do and just comes to look for details. The latter is tutorial - a person does not know how to achieve something, thus one reads tutorial start to end to get general understanding.

In this particular case I think we should mention that there's a better way to do things in both places. I agree that this introduces some duplication, but it serves a practical purpose that developers reading the documentation are less confused. I think this saves time overall if we think about both developers of this project and its users.

Finally, reducing duplication serves the purpose of reducing the chances of errors, when only one repetition is updated. I think this is less of concern in this case, because the next change is likely to be removal of inlineCallbacks some years into the future.